kernel/generic/src/ipc/irq.c

```diff
@@ -294,15 +294,15 @@ static void irq_hash_out(irq_t *irq)
 	irq_spinlock_unlock(&irq_uspace_hash_table_lock, true);
 }
 
-static void irq_destroy(void *arg)
+static void irq_destroy(kobject_t *arg)
 {
-	irq_t *irq = (irq_t *) arg;
+	irq_kobject_t *kobj = (irq_kobject_t *) arg;
```

Comment:
For the sake of consistency, irq_from_kobject(arg)?

Reply:
I think also in other places...

Reply:
Yeah, probably. I omitted it here because it's all in the same file, but it makes sense to make it consistent.